pyproject.toml

```diff
@@ -259,3 +257,10 @@ flat_sectorfile_parser = "geoips.interface_modules.trackfile_parsers.flat_sector
 static_standard = "geoips.plugins.modules.title_formatters.static_standard"
 tc_copyright = "geoips.plugins.modules.title_formatters.tc_copyright"
 tc_standard = "geoips.plugins.modules.title_formatters.tc_standard"
+
+[tool.poetry.plugins."geoips.output_checkers"]
+geotiff_checker = "geoips.plugins.modules.output_checkers.geotiff_checker"
+gz_checker = "geoips.plugins.modules.output_checkers.gz_checker"
+image_checker = "geoips.plugins.modules.output_checkers.image_checker"
+netcdf_checker = "geoips.plugins.modules.output_checkers.netcdf_checker"
+text_checker = "geoips.plugins.modules.output_checkers.text_checker"
```

Comment:
This is the only actual change to pyproject.toml, correct?

Reply:
Yup. Not sure why there are other things included in this diff. This is the only change I made to pyproject.toml